registry/storage/shares.go

```diff
@@ -58,6 +58,7 @@ type sharesStorage struct {
 	shares         map[string]*types.SSVShare
 	validatorStore *validatorStore
 	mu             sync.RWMutex
+	dbmu           sync.Mutex
```

Comment:
Would be nice to have some comment for each of these mutex(s) clarifying how they are different.